scripts/helpers/strip.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+/**
+ *
+ * @param {string} path
+ */
+module.exports = function(path, options) {
+    return path.replace('/index.html', "")
```

Comment:
are you sure you wanted to use double quotes here?

Reply:
Maybe i was daydreaming :). You have very strong attention to detail. I don't know why my tools didn't catch that. :(